hare/eligibility/oracle.go

```diff
@@ -159,3 +204,52 @@ func (o *Oracle) Proof(id types.NodeId, layer types.LayerID, round int32) ([]byt
 
 	return sig, nil
 }
+
+// Returns a map of all active nodes in the specified layer id
+func (o *Oracle) actives(layer types.LayerID) (map[string]struct{}, error) {
+	sl := roundedSafeLayer(layer, k, o.layersPerEpoch, constOffset)
+
+	ep := sl.GetEpoch(o.layersPerEpoch)
+	// check genesis
+	if ep.IsGenesis() {
+		return nil, genesisErr
+	}
+
+	// check cache
+	if val, exist := o.cache.Get(sl); exist {
+		return val.(map[string]struct{}), nil
+	}
+
+	activeMap, err := o.getActiveSet(sl, o.layersPerEpoch)
+	if err != nil {
+		o.With().Error("Could not retrieve active set size", log.String("err", err.Error()),
+			log.Uint64("layer", uint64(layer)), log.Uint64("epoch", uint64(layer.GetEpoch(o.layersPerEpoch))),
```

Comment:
change the key "layer" to "layer_id" and "epoch" to "epoch_id"

Reply:
Fixed